src/cryptoClients/ecdsaSigFormatter.js

```diff
@@ -1,3 +1,13 @@
+/*
+ * This code is taken from https://github.com/Brightspace/node-ecdsa-sig-formatter
+ * which is licensed under the Apache 2.0 license.
+ * 
+ * It got copied over here to make some adjustments for being compatible with browserify.
+ * Going forward would be either simplifying this code (as we only need 256 bit signatures),
+ * or moving back to the direct dependency; both is future work™ for some other day.
```

Comment:
can we replace the TM with (TM)? I'd prefer ascii only for now in the source code :smile:

Reply:
Sure :sweat_smile: